eng/pipelines/mgmt.yml

```
@@ -202,7 +202,7 @@ stages:
           Scope: ${{ parameters.Scope }}
           ShouldPublish: ${{ parameters.ShouldPublish }}
 
-      - ${{ if and(ne(variables['Build.Reason'], 'PullRequest'), eq(parameters.ShouldPublish, 'true'),  eq(variables['System.TeamProject'], 'internal')) }}:
+      - ${{ if and(eq(parameters.ShouldPublish, 'true'), ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'), eq(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-net')) }}:
```

Just to help with readability can you please nest the conditions similar to https://github.com/Azure/azure-sdk-for-go/blob/main/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L194.

```suggestion
      - ${{ if and(eq(parameters.ShouldPublish, 'true'), eq(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-net')) }}
        - ${{ if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal')) }}:
```
need to indent the lines under the condition one more level.